crates/re_arrow_store/src/store_stats.rs

```diff
@@ -429,17 +422,22 @@ impl SizeBytes for PersistentIndexedTable {
         let Self {
             ent_path,
             cluster_key,
+            inner,
+        } = self;
+        let PersistentIndexedTableInner {
             col_insert_id,
             col_row_id,
             col_num_instances,
             columns,
-        } = self;
+            is_sorted,
+        } = &*inner.read();
 
         ent_path.total_size_bytes()
             + cluster_key.total_size_bytes()
             + col_insert_id.total_size_bytes()
             + col_row_id.total_size_bytes()
             + col_num_instances.total_size_bytes()
             + columns.total_size_bytes()
+            + is_sorted.total_size_bytes()
```

Comment:
offtopic, but this way of counting the size of a struct (and what it points to) isn't all that accurate since padding between fields can play a big role :/. Not that it matters all that much in the big picture here

Reply:
Yep, the stats are not meant to be perfectly accurate, they just need to be "good enough" and more importantly _consistent_: since the GC works with percentages rather than raw numbers, as long as things are consistent it works out okay.

By far our biggest problem at the moment is the size of serialized arrow data, which is both wrong and inconsistent in many ways lol.

Quoting the docs:
```rust
/// The size of both the control & component data stored in this bucket, heap and stack
/// included, in bytes.
///
/// This is a best-effort approximation, adequate for most purposes (stats,
/// triggering GCs, …).
```